### DISCUSSION TITLE
Release/5.9.5

### DISCUSSION THREAD

**Discussion by Rixxan** [EDCD/EDMarketConnector#2066] [Announcement]
Release 5.9.5
===
This release fixes an uncommon problem with the uninstaller logic if upgrading from a version prior to 5.9.0 to improve consistancy across versions.

_Note to plugin developers: modules.p and ships.p will be deprecated in the next version, and slated for removal in the next major release! Please look for that change coming soon._ 

- Updates Module pickle files to latest values.
- Fixes a problem with the uninstaller logic caused by prior versions having fluctuating GUIDs.

<hr /><em>This discussion was created from the release <a href='https://github.com/EDCD/EDMarketConnector/releases/tag/Release/5.9.5'>Release/5.9.5</a>.</em>

**Comment by WheresJoe** [comment]
Just an idea, to add to this fantastic Add-in - How they are built ais way beyond my PC prowess, but would it be possible in the display area to show the current financial holdings - i.e how much cash is in my Balance displayed on board RH Screen?
Again such a useful tool, i love it 
many, thanks

Joe o7
